### PR TITLE
Box integration

### PR DESCRIPTION
**Pull request by Spivaka**
fixes https://github.com/demisto/server/issues/4271

### CODE REVIEW

**Comment by Spivaka** [comment]
With the new feature for persisting state and retries to avoid collisions

**Comment by aviadl** [comment]
Please run a concurrent test 
Will check the code later

**Comment by Spivaka** [comment]
Lets discuss tomorrow - as "concurrent test" is not very well defined

**Comment by Spivaka** [comment]
@aviadl please review and assig @liorkol to write scripts and use cases for it

File: Integrations/integration-Box.yml [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
...m+pq3l5uBfi8IaGrD6x47Oe+Nep4mrw//YT7RGVu1T1znhD6F3XMUSlIBxPnWNEf6iQxa5biPLAZhX56aGyfau5sql4hePbjhUxGwB83nVLAXoH1p4bb5i6zXUIUSn5cDuXVg4Bh6RPVRYmItW/y8XFOACkYYu09FTDZL4L4FoAk1z3EI0mAXoVuMvLBH7Ot8lEboTj6fli9AYoFgEwrnuIRpMAL1rVm3vLQw89dZ5kXffQyAvHUyeJ4GkAx7puKSA7PZH5zfXVr7sOISpFi9bs+ouxfX2/BfAl1y0FREX16pZIzapcXZADQMqZaGxTWbZs0qWiegOAWa57iEbYb1XlF4Hs7rbmxpn9rmOICFjS9l7IU2+ZqHwHPDmYilsWwIOwuC3eEHrJdQyNvmh71wzft48D+KzrlgKw2xezoK2+aqPrEKJS9uEQ8AkAp7tuKQB9AlzaEg615vKiHABSzjU1qXl1Troegu8KMB/8PaPi0QcgBmvu4Im+RPmrtiM1LpjVywD5FoAvuu4hyqHdInjAWtyViIQ2u44htzgEHJQdYrC4pS70gusQIgJqO1KTg1lpB/Qs1y15S3BIrV6aiNS05f7SRCOo4aGtJ6jnXQFgOYBqxzlEQ/WOit4V6Dermhure1zHENHgRVq6zlRjvy1AFMAY1z1EQ6CArhPgngnv9yfuu2LGIddBlD+ise4qG8w+psBs1y35RoE3Ap5Z1LykKum6hYj+JBrbOtYPeL+CIOy6JQ/thZG6eF31upG4OAeANCoWrNPAsXtSi9SaqyF6MYCA6yaiwxIcUkXCiK6a/UroyaYmsa6TiGjoorGtk/yguRyQKwDMdd1D9OkkBdX7rHirWsOVW1zXUP6KxnYfY4OHVikQcd2SRzao2iWJyNRdrkOI6OOamtRsPDV9qyquc92SLxR4A2qWJiJVr43UPTgApFEXaempVq//r0XlSvAEM8ozAryoIqv6Av0PPrz4uF7XPUSUew1t2+eqNVcAejl4eBXllywgj4jY/9h1TOgRHupBg6YqDYn091TwUwBB1zlOKe7xsv51zY3TDrpOIaLDC8dTfyOCWwCMc93ikgiaMxK8qr2uYu+I3mckL050WKoSaU+fCx9XQRABcLTrJCpZ3Qo8aIy/qqVuWqfrGCIaHYvWvDVmbP+4ekCWQ/FVAJ7rJipRKp0Q/XXWZP+zve4zK
```
Comment (aviadl) [inline Integrations/integration-Box.yml]: LInting :)
Add space after colon
and in the row bellow

Comment (aviadl) [inline Integrations/integration-Box.yml]: linting see above

Comment (aviadl) [inline Integrations/integration-Box.yml]: Dont we need here wait
And if so, it should probably be with a random time

**Comment by Spivaka** [comment]
@aviadl all your comments are on the same image line... Not sure what to fix

File: Integrations/integration-Box.yml (hunk 1 above)
Comment (aviadl) [inline Integrations/integration-Box.yml]: LInting

Comment (aviadl) [inline Integrations/integration-Box.yml]: linting

Comment (aviadl) [inline Integrations/integration-Box.yml]: sleep

**Comment by Spivaka** [comment]
Still all on the image... lets review on sunday f2f

**Pull request merged**